lib/bloc/account/account_model.dart

```diff
@@ -218,6 +224,8 @@ class AccountModel {
       (f) => f.currencyData.shortName == _fiatShortName,
       orElse: () => null);
   List<FiatConversion> get fiatConversionList => _fiatConversionList;
+  String get posCurrencyShortName => _posCurrencyShortName;
+  CurrencyWrapper get posCurrency => CurrencyWrapper.fromShortName(_posCurrencyShortName, this);
```

Comment:
The CurrencyWrapper is still a UI element, we can't use it in the bloc.
Later when the pos stabilizes I think we should create a unified interface for btc currencies and fiat currencies.

Reply:
[Done](https://github.com/breez/breezmobile/pull/309/commits/c421d202e4f3bf9da390a0f2adea75cd720eadfa)
I agree.